framework/data/Sort.php

```diff
@@ -422,4 +422,28 @@ public function hasAttribute($name)
     {
         return isset($this->attributes[$name]);
     }
+
+    /**
+     * Parses [[sortParam]] into an array of sort attributes.
+     *
+     * The format must be:
+     * - Ascending: only attribute name, e.g. 'category'
+     * - Descending: attribute name is prefixed with `-`, e.g. '-category'
+     *
+     * The following return value will result in ascending sort by `category` and descending sort by `created_at`:
+     *
+     * ```php
+     * [
+     *     'category',
+     *     '-created_at'
+     * ]
+     * ```
+     * @param array $params the value of the [[sortParam]].
+     * @return array the sort attributes.
```

Comment:
I don't understand. Is only add `@see separator` on doc block?
like: 
```php
    /**
     * Parses [[sortParam]] into an array of sort attributes.
     *
     * The format must be:
     * - Ascending: only attribute name, e.g. 'category'
     * - Descending: attribute name is prefixed with `-`, e.g. '-category'
     *
     * The following return value will result in ascending sort by `category` and descending sort by `created_at`:
     *
     * ```php
     * [
     *     'category',
     *     '-created_at'
     * ]
     * ```
     * @param array $params the value of the [[sortParam]].
     * @return array the sort attributes.
     * @since 2.0.11
     * @see separator
     */
```